specs/latest/2.0/index.html

```diff
@@ -1092,6 +1094,52 @@ <h2 class="no-toc">Table of contents</h2>
         it is the responsibility of the WebGL API to ensure that data are accessed consistently. This applies
         even if the buffer is currently bound to a transform feedback binding point.
       </dd>
+      <dt>
+        <p class="idl-code">Promise&lt;ArrayBuffer&gt; getBufferSubDataAsync(GLenum target, GLintptr offset, ArrayBuffer returnedData)
```

Comment:
I am not sure that passing in "returnedData" is the best idea for the async version of this API. This can easily lead to programming errors such as race conditions (apps touching returnedData before the promise has resolved) and memory leaks (apps not handling promise rejection adequately).  What are the downsides of having this API allocate its own ArrayBuffer?